website/docs/releases.md

```diff
@@ -28,6 +28,10 @@ In case you are wondering why all our releases start with `0.0`, read [this FAQ
 
 ### Upcoming Release
 
+* We have introduced a new 32-bit floating point data type.
+  * Persisting it is only supported in the new file format 4.
+  * A `CAST(… AS double precision)` is needed to store such columns in older file formats.
+* We announced file format 3 (bigger 128-bit numerics) and file format 4 (32-bit floating points)
```

Comment:
Pleas split into two bullet points. One for format 3, one for format 4

Reply:
0ff798c4877d54afde264ea438cc3885b320eb87